pkg/tagger/local/tagger.go

```diff
@@ -272,9 +273,26 @@ IterCollectors:
 	}
 	t.RUnlock()
 
-	computedTags := utils.ConcatenateTags(tagArrays)
+	return utils.ConcatenateTags(tagArrays), nil
+}
+
+// TagBuilder appends tags for a given entity from the tagger to the TagsBuilder
+func (t *Tagger) TagBuilder(entity string, cardinality collectors.TagCardinality, tb *util.TagsBuilder) error {
```

Comment:
Maybe it can be good to have an interface for the TagsBuilder to ease unit-test.

```golang
type TagsBuilderClientInterface interface {
    Append(tags ...string)
}

Reply:
Not sure what it would help with. TagsBuilder is such a basic class in the utils that you don't really need to mock it, no ?

Reply:
Only if we want to try different implementations :)

Reply:
It's so basic that I doubt we'd want one (I can't come up with a use case). Also if one day we find one, it's so simple that it will be trivial to do. In the meantime I'd rather have a basic/simple code.